lib/superstore/adapters/jsonb_adapter.rb

```diff
@@ -43,48 +48,31 @@ def update(table, id, attributes)
         nil_properties = attributes.each_key.select { |k| attributes[k].nil? }
         not_nil_attributes = attributes.reject { |key, value| value.nil? }
 
+        statement = Arel::UpdateManager.new
+        statement.table(Arel::Table.new(table))
+        statement.where(Arel::Nodes::SqlLiteral.new(primary_key_column).eq(id))
+
         value_update = superstore_column
         nil_properties.each do |property|
           value_update = "(#{value_update} - '#{property}')"
         end
 
         if not_nil_attributes.any?
-          value_update = "(#{value_update} || #{to_quoted_jsonb(not_nil_attributes)})"
+          value_update = "(#{value_update} || #{quote(to_quoted_jsonb(not_nil_attributes))})"
```

Comment:
It's a little confusing that `quote` is necessary here when the method name is already `to_quoted_jsonb`.

Reply:
Renamed